c/check.py

```diff
@@ -85,6 +86,89 @@
     ]
 
 KNOWN_BENCHMARK_FILE_PROBLEMS = [
```

Comment:
Would it be possible to right away open an issue (with checklist) to track those?

Reply:
I was planning to do so when this PR gets merged.